app/routes/docs/index.tsx

```diff
@@ -0,0 +1,12 @@
+/*!
+ * Copyright © 2022 United States Government as represented by the Administrator
+ * of the National Aeronautics and Space Administration. No copyright is claimed
+ * in the United States under Title 17, U.S. Code. All Other Rights Reserved.
+ *
+ * SPDX-License-Identifier: NASA-1.3
+ */
+export { Outlet as default } from 'react-router'
+
+export const handle = {
+  breadcrumb: 'Documentation',
+}
```

Comment:
Since the layout route app/routes/docs.tsx already exports a breadcrumb `'Documentation'`, this would be rendered as `GCN - Documentation - Documentation`.

```suggestion
```